designer/server/src/main/scala/pl/touk/nussknacker/ui/process/test/TestResultsJsonEncoder.scala

```diff
@@ -0,0 +1,66 @@
+package pl.touk.nussknacker.ui.process.test
+
+import io.circe.generic.extras.semiauto.deriveConfiguredEncoder
+import io.circe.{Encoder, Json}
+import pl.touk.nussknacker.engine.api.DisplayJson
+import pl.touk.nussknacker.engine.testmode.TestProcess._
+import pl.touk.nussknacker.engine.util.json.BestEffortJsonEncoder
+
+object TestResultsJsonEncoder {
```

Comment:
Hmm, it's ManagementResources specific, maybe move it do designer?

Reply:
What do you mean by moving it to Designer. I see that it is already there. I would even move it close to the `ManagementResources` class, maybe into companion object. WDYT?

Reply:
Done